SwiftWisdomTests/StandardLibrary/ArrayUtilitiesTests.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  Array+UtilitiesTests.swift
+//  SwiftWisdom
+//
+//  Created by Alexander Persian on 9/30/16.
+//  Copyright © 2016 Intrepid. All rights reserved.
+//
+
+import XCTest
+import SwiftiWisdom
+
+class ArrayUtilities: XCTestCase {
+
+    func testSubArrayFromIndicesOf() {
+        let testArray = [4, 8, 15, 16, 23, 42]
+        let indices = testArray.ip_indicesOf(elements: [15, 16, 19])
+        let subTestArray = testArray.ip_subArrayFrom(indices: indices)
+        XCTAssert(subTestArray == [15, 16])
+    }
+
+    func testRemoveObject() {
+        var testArray = [1, 5, 9, 14, 12]
+        var result = testArray.ip_remove(object: 9)
+        XCTAssertTrue(result)
+        XCTAssert(testArray = [1, 5, 14, 12])
+
+        result = testArray.ip_remove(object: 42)
+        XCTAssertFalse(result)
+        XCTAssert(testArray = [1, 5, 14, 12])
+    }
+
+    func testRemoveElements() {
+        var testArray = [1, 5, 9, 14, 12]
+        testArray.ip_remove(elements: [1, 5, 9])
+        XCTAssert(testArray == [14, 12])
+        testArray.ip_remove(elements: [12])
+        XCTAssert(testArray == [14])
+    }
```

Comment:
maybe include a case where you try to remove elements that are not present in testArray like you are with `testRemoveObject()` above?